amqp/src/test/java/akka/stream/alpakka/amqp/javadsl/AmqpConnectorsTest.java

```diff
@@ -130,7 +131,7 @@ public void publishAndConsumeRpc() throws Exception {
     );
 
     amqpSource.map(b ->
-        new OutgoingMessage(b.bytes().concat(ByteString.fromString("a")), false, false, Some.apply(b.properties()))
+        new OutgoingMessage(b.bytes().concat(ByteString.fromString("a")), false, false, Some.apply(b.properties()), Option.apply(null))
```

Comment:
Can we provide a Java API here? I can see that it was already missing but a constructor taking `Optional` instead of `Option` would be nice.